src/resources/page-names.yml

```diff
@@ -13,6 +13,9 @@ settings:
 import:
   en: Import
   de: Import
+imports:
+  en: Imports overview
+  de: Übersicht über Importe
```

Comment:
This would sound to me more like `Import-Übersich` or `Importsübersicht`, but of course the native has the call here 😋

Reply:
I think `Import-Übersicht` sounds best.

Reply:
Okay, will do, google translate lied to me!

Reply:
No, but GT would not pass a C level exam

Reply:
Fixed :D